Samples/WinMLSamplesGallery/WinMLSamplesGalleryNative/WinMLSamplesGalleryNative.vcxproj

```diff
@@ -111,6 +112,10 @@
     <UseOpenCV>False</UseOpenCV>
     <UseOpenCV Condition="Exists('$(OpenCVLibFullPath)')">True</UseOpenCV>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
+    <IncludePath>C:\Program Files %28x86%29\Windows Kits\10\Include\10.0.19041.0\um;C:\Program Files %28x86%29\Windows Kits\10\Include\10.0.19041.0\shared;$(IncludePath)</IncludePath>

```

Comment:
><IncludePath>C:\Program Files %28x86%29\Windows Kits\10\Include\10.0.19041.0\um;C:\Program Files %28x86%29\Windows Kits\10\Include\10.0.19041.0\shared;$(IncludePath)</IncludePath> [](http://example.com/codeflow?start=4&length=179)

Hmm, full paths shouldn't be in the vcxproj, as these won't work on other machines

Reply:
Yes, removed.